build.gradle

```diff
@@ -38,11 +38,16 @@ gradlePlugin {
 
     automatedPublishing = true
 
+    def releaseNotes = layout.projectDirectory.file('release/changes.md').asFile.text
     plugins {
         commonCustomUserData {
             id = 'com.gradle.common-custom-user-data-gradle-plugin'
             displayName = 'Gradle Enterprise Common Custom User Data Gradle Plugin'
-            description = 'A Gradle plugin to capture common custom user data used for Gradle Build Scans in Gradle Enterprise'
+            description = """\
+                          |A Gradle plugin to capture common custom user data used for Gradle Build Scans in Gradle Enterprise
+                          |
+                          |$releaseNotes
+                          """.stripMargin().trim()
```

Comment:
I recommend doing two things. 

- Leverage `project.description` for the first bit. Right now we don't have a project description so that would be a nice enhancement anyway. Then in this logic you can replace hard-coded text with `project.description`. 
- Encapsulate building the release notes to a function so that it can more easily be brought over to other projects.

Reply:
We could also scrap adding the `project.description` to the release notes entirely. The [`com.gradle.plugin-publish`](https://plugins.gradle.org/plugin/com.gradle.plugin-publish) does not include it.

Reply:
> We could also scrap adding the project.description to the release notes entirely. The [com.gradle.plugin-publish](https://plugins.gradle.org/plugin/com.gradle.plugin-publish) does not include it.

Yeah I think let's follow the example of the `plugin-publish` plugin. It already links to the repo on GitHub anyways, so you're a click away from the README.